translations/fr/_layouts/sidebar.md

```diff
@@ -5,6 +5,7 @@
   - [Outils pour réussir](/translations/fr/04-outils-pour-reussir.md)
   - [Comment contribuer à l'open source](/translations/fr/05-comment-contribuer-à-l-open-source.md)
   - [La sauce secrète](/translations/fr/06-la-sauce-secrète.md)
-  - [Conclusion](/translations/fr/07-conclusion.md)
+  - [Les types de contributions](/translations/fr/07-les-types-de-contributions.md)
   - [Ressources Additionnelles](/translations/fr/08-ressources-additionnelles.md)
-  - [Glossaire](/translations/fr/09-glossaire.md)
+  - [Conclusion](/translations/fr/09-conclusion.md)
+  - [Glossaire](/translations/fr/10-glossaire.md)
```

Comment:
IMHO, whatever change is related to French translation, there should be another PR.